egs/gigaspeech/ASR/zipformer/asr_datamodule.py

```diff
@@ -0,0 +1,444 @@
+# Copyright      2021  Piotr Żelasko
```

Comment:
Is it possible to replace it with a symlnik?

Reply:
I have modified this file. In the past, GigaSpeech XL splits would be merged. This one uses `lhotse.mux`.